lib/rucio/api/rse.py

```diff
@@ -243,17 +244,18 @@ def set_rse_usage(rse, source, used, free, issuer):
     return rse_module.set_rse_usage(rse=rse, source=source, used=used, free=free)
 
 
-def get_rse_usage(rse, issuer, source=None):
+def get_rse_usage(rse, issuer, source=None, per_account=False):
     """
     get RSE usage information.
 
     :param rse: The RSE name.
     :param issuer: The issuer account.
     :param source: dictionary of attributes by which the results should be filtered
+    :param per_account: Boolean whether usage should be calculated per account or not
 
     :returns: True if successful, otherwise false.
```

Comment:
I know you did not write this line, but please change to what is actually happening.

Reply:
I mean the `:returns: True if successful, otherwise false.` should actually be a `:returns: List of RSE usage data`